src/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    env: {
+        browser: true,
+    },
+    extends: [
+    	"plugin:compat/recommended"
```

Comment:
Why did you create another eslintrc?

Reply:
So that /src is linted separately from /tests.
Tests don't require browser compatibility.

Reply:
Oh, I see now. Thnx